exercises/custom-converter/practice/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "durable-execution",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "lint": "eslint .",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "workflow": "ts-node src/client.ts",
+    "service": "ts-node src/service.ts",
+    "service.watch": "nodemon src/service.ts",
+    "format": "prettier --config .prettierrc 'src/**/*.ts' --write"
+  },
+  "nodemonConfig": {
+    "execMap": {
+      "ts": "ts-node"
+    },
+    "ext": "ts",
+    "watch": [
+      "src"
+    ]
+  },
+  "dependencies": {
+    "@temporalio/activity": "~1.8.0",
+    "@temporalio/client": "~1.8.0",
+    "@temporalio/worker": "~1.8.0",
+    "@temporalio/workflow": "~1.8.0",
+    "axios": "^1.4.0",
+    "ejson": "^2.2.3",
+    "express": "^4.18.2",
+    "nanoid": "3.x",
+    "uuid": "^9.0.1"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "^1.0.0",
```

Comment:
Unless there's a strong reason not to do so, I'd personally push for all education content to be based on the current LTS release of Node, which is Node 20 at this time. I mean, somebody who would be _starting_ a project today, after going through that exercise, should go for the active LTS version of Node. We don't want to encourage people to use a release of Node that is about to reach end of line.

Reply:
Thanks, have done this and made a ticket for myself to do this for all courses: https://temporalio.atlassian.net/browse/EDU-2282